ci/pipelines/shared/jinja.variables.yml

```diff
@@ -135,6 +135,18 @@ tests:
   PLATFORM: linux
   RAM: '210'
   name: Upgrade
+- ARTIFACT_SLUG: concurrencytestfiles
+  CALL_STACK_TIMEOUT: '1200'
+  CPUS: '96'
+  DUNIT_PARALLEL_FORKS: '0'
+  EXECUTE_TEST_TIMEOUT: 30m
```

Comment:
Since we turned off parallel everything, is 30 minutes too short for these tests?

Reply:
They finish in under 10 minutes.